lib/dash/content_protection.js

```diff
@@ -182,7 +182,7 @@ shaka.dash.ContentProtection = class {
 
   /**
    * Gets a ClearKey license URL from a content protection element
-   * containing a custom `clearkey::Laurl` element
+   * containing a custom `clearkey::Laurl` or 'dashif:Laurl' elements
```

Comment:
I don't think this is correct.  The user's example is ClearKey, but dashif:Laurl can be used with any DRM system (based on the schemeIdUri in the enclosing ContentProtection element).

Also, the dashif docs state that if you use dashif:Laurl, you also have to support dashif:authzurl.  Details here:

 - https://dashif-documents.azurewebsites.net/Guidelines-Security/master/Guidelines-Security.html#CPS-mpd-drm-config
 - https://dashif-documents.azurewebsites.net/Guidelines-Security/master/Guidelines-Security.html#CPS-lr-model
 - https://dashif-documents.azurewebsites.net/Guidelines-Security/master/Guidelines-Security.html#CPS-license-request-workflow

Reply:
I also added support for Widevine and Playready.

About dashif:authzurl (https://dashif-documents.azurewebsites.net/Guidelines-Security/master/Guidelines-Security.html#CPS-lr-model-authz):
`Attaching this proof of authorization to a license request is optional`

My use case does not require this part yet. It is also a big change compared to what we have today, since this request is not included in the data model. My proposal is to leave this optional part as an improvement for the future, is that okay with you?

Reply:
Sounds good.